### PR TITLE
chore(chart)!: update default cometbft config vals

### PR DESCRIPTION
**Pull request by joroshiba**
## Summary
Updates default CometBFT configuration, and genesis.

## Background
We have been running on mostly defaults without much thought to it, gave a more thorough review.

## Breaking Changelist
- Changes genesis parameters and thus cannot be used with existing networks without adding ability to make more configurable w/ these as defaults.

### CODE REVIEW

File: charts/sequencer/files/cometbft/config/config.toml [hunk 1]
```diff
@@ -297,15 +297,15 @@ broadcast = true
...
 
 # Maximum number of transactions in the mempool
-size = 5000
+size = 2000
 
 # Limit the total size of all txs in the mempool.
 # This only accounts for raw transactions (e.g. given 1MB transactions and
 # max_txs_bytes=5MB, mempool will only accept 5 transactions).
-max_txs_bytes = 1073741824
+max_txs_bytes = 100_000_000
```
Comment (noot) [inline charts/sequencer/files/cometbft/config/config.toml:305]: why the change here?

File: charts/sequencer/files/cometbft/config/genesis.json [hunk 2]
```diff
@@ -46,12 +46,12 @@
...
     "block": {
-      "max_bytes": "22020096",
+      "max_bytes": "1000000",
       "max_gas": "-1"
     },
     "evidence": {
-      "max_age_duration": "172800000000000",
-      "max_age_num_blocks": "100000",
+      "max_age_duration": "1209600000000000",
+      "max_age_num_blocks": "4000000",
```
Comment (noot) [inline charts/sequencer/files/cometbft/config/genesis.json:54]: same here

File: charts/sequencer/files/cometbft/config/config.toml (hunk 1 above)
Comment (quasystaty1) [reply charts/sequencer/files/cometbft/config/config.toml:305]: Decreasing mempool size from 1GB to 100 MB, better default configs, same for other values.

**Comment by joroshiba** [comment]
Fixed as a part of  https://github.com/astriaorg/astria/pull/1193
